README.md

```diff
@@ -39,24 +52,82 @@ for msg := range sub.MessageChannel() {
 ### Publishing to a channel
 
 ```go
-res, err := channel.Publish("EventName1", "EventData2")
+// send request to a server
+res, err := channel.Publish("EventName1", "EventData1")
+if err != nil {
+	panic(err)
+}
+
+// await confirmation
+if err = res.Wait(); err != nil {
+	panic(err)
+}
+```
+
+### Announcing presence on a channel
+
+```go
+// send request to a server
+res, err := channel.Presence.Enter("presence data")
 if err != nil {
 	panic(err)
 }
 
-fmt.Println("message sent successfully, awaiting confirmation. . .")
+// await confirmation
+if err = res.Wait(); err != nil {
+	panic(err)
+}
+```
+
+### Announcing presence on a channel on behalf of other client
 
+```go
+// send request to a server
+res, err := channel.Presence.EnterClient("clientID", "presence data")
+if err != nil {
+	panic(err)
+}
+
+// await confirmation
 if err = res.Wait(); err != nil {
 	panic(err)
 }
+```
 
-fmt.Println("message published successfully")
+### Getting all clients present on a channel
+
+```go
+clients := channel.Presence.Get(true)
```

Comment:
This does not quite match our API, see http://www.rubydoc.info/gems/ably/0.8.2/Ably/Realtime/Presence#get-instance_method for an example.  We should be able to provide options, one of which is the `waitForSync` boolean